sale_coupon_multi_gift/models/sale_coupon_reward.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, fields, models
+
+
+class SaleCouponReward(models.Model):
+    _inherit = "sale.coupon.reward"
+
+    coupon_multi_gift_ids = fields.One2many(
+        comodel_name="sale.coupon.reward.product_line",
+        inverse_name="reward_id",
+        string="Gift list",
+    )
+    discount_line_product_id = fields.Many2one(
```

Comment:
Why do you need this?

Reply:
Oops, I don't know where it came from. Removing it